Frontend/ui-customer/src/Resources/Localizable/Types.purs

```diff
@@ -597,7 +597,8 @@ data STR
  | AC_TAXI 
  | NON_AC_TAXI
  | GET_OTP_VIA_WHATSAPP
- | OR
+ | OR | AUTO_RICKSHAW
```

Comment:
@Anuragini-2506 please correct the indentation here.

Reply:
done